theme.html

```diff
@@ -0,0 +1,284 @@
+            <li class="nav-item active">
+               <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
+            </li>
+            <li class="nav-item">
+               <a class="nav-link" href="#about">About</a>
+            </li>
+           <!--  <li class="nav-item">
+               <a class="nav-link" href="#register">Registration</a>
+            </li> -->
+            <!-- <li class="nav-item">
+               <a class="nav-link" href="resources.html">Resources</a>
+            </li> -->
+            <!-- <li class="nav-item">
+               <a class="nav-link" href="submissions.html">Submissions</a>
+            </li> -->
+            <!-- <li class="nav-item">
+               <a href="results.html" class="nav-link">Results</a>
+            </li> -->
+             <!-- <li class="nav-item">
+               <a class="nav-link" href="#">Schedule</a>
+            </li> -->
+            <li class="nav-item">
+               <a class="nav-link" href="#endorsement-partners">Endorsement Partners</a>
+            </li>
+            <li class="nav-item">
+               <a class="nav-link" href="#partners-and-organisers">Partners and Organisers</a>
+            </li>
+            <!-- <li class="nav-item">
+               <a class="nav-link" href="#rules-and-regulations">Rules and Regulations</a>
+            </li> -->
+             <li class="nav-item">
+               <a class="nav-link" href="#">Rewards</a>
+            </li>
+            <li class="nav-item">
+               <a class="nav-link" href="#faqs">FAQs</a>
+            </li>
+            <li class="nav-item">
+               <a class="nav-link" href="#contact-us">Contact us</a>
+            </li>
+         </ul>
+      </div>
+   </nav> 
+   <br>
+ <h2><center><p style="color: purple" id="demo"></p></center></h2>
+
+<script>
+// Set the date we're counting down to
+var countDownDate = new Date("Nov 25, 2020 00:00:00").getTime();
+
+// Update the count down every 1 second
+var x = setInterval(function() {
+
+  // Get today's date and time
+  var now = new Date().getTime();
+    
+  // Find the distance between now and the count down date
+  var distance = countDownDate - now;
+    
+  // Time calculations for days, hours, minutes and seconds
+  var days = Math.floor(distance / (1000 * 60 * 60 * 24));
+  var hours = Math.floor((distance % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60));
+  var minutes = Math.floor((distance % (1000 * 60 * 60)) / (1000 * 60));
+  var seconds = Math.floor((distance % (1000 * 60)) / 1000);
+    
+  // Output the result in an element with id="demo"
+  document.getElementById("demo").innerHTML = days + "d " + hours + "h "
+  + minutes + "m " + seconds + "s ";
+    
+  // If the count down is over, write some text 
+  if (distance < 0) {
+    clearInterval(x);
+    document.getElementById("demo").innerHTML = "EXPIRED";
+  }
+}, 1000);
+</script>
+
+  <section id="results">
```

Comment:
Remove bg-light class applied to the div

Reply:
Done